org.eclipse.m2e.launching/src/org/eclipse/m2e/internal/launch/ColorOutput.java

```diff
@@ -0,0 +1,35 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Mihai Nita
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *      Mihai Nita - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.m2e.internal.launch;
+
+/**
+ * Named constants for the various color settings passed to Maven.
+ */
+public class ColorOutput {
```

Comment:
I think this could better be placed in MavenLaunchConstants, e.g.

ATTR_COLOR_AUTO = 0
ATTR_COLOR_ALWAYS = 1
ATTR_COLOR_NEVER = 2

Reply:
Done :-)